Plugins/BTCPayServer.Plugins.ShopifyPlugin/Views/UIShopify/ShopifyAdmin.cshtml

```diff
@@ -14,4 +14,10 @@
 <header class="text-center">
     <h1>Shopify Plugin for BTCPay Server</h1>
     <p class="lead text-secondary mt-3">Please, <a href="@url" target="_blank">click here</a> to go to the plugin's page</p>
+
```

Comment:
this does not look right :laughing: 

![image](https://github.com/user-attachments/assets/ee6d958b-61e3-4ffe-89d6-7b3516a263dd)

Reply:
Not sure what the intention was if only the duplicated headline is the issue. I would leave the link to plugin page but your additional information is helpful :+1: